forseti-api/README.md

```diff
@@ -1,47 +1,37 @@
 # Forseti API
 
-The node.js backend for the forseti-visualizer project.  Forseti API serves content from the forseti_security Cloud SQL database and the GRPC server on the forseti-server vm.
+The node.js backend for the forseti-visualizer project. Forseti API serves content from the forseti_security Cloud SQL database and the GRPC server on the forseti-server vm.
 
 ## Pre-Requisites
 
-* A database connection using [Cloud SQL Proxy (recommended)](#Cloud-SQL-Proxy).
-* If using IAM Explainer, the hosted solution must have access to the forseti-server GCE VM via a Firewall enabling GRPC traffic (port 50051)
+- Follow the instructions FIRST on the forseti-visualizer-ui/README.md.
+- A database connection using [Cloud SQL Proxy (recommended)](#Cloud-SQL-Proxy).
+- If using IAM Explainer, the hosted solution must have access to the forseti-server GCE VM via a Firewall enabling GRPC traffic (port 50051)
 
 To build and deploy the solution from your local system, please ensure you have the following:
 
-* docker 18.05+
-* kubectl 1.9.7++
+- docker 18.05+
+- kubectl 1.9.7++
 
 ## Getting Started - Development Locally
 
 To get started, you will need to create a local source.env file and source the file.
 
-* Handle requires a Data Model to be created (forseti create model)
-* Query the handle using `SELECT handle FROM forseti_security.model order by created_at_datetime desc LIMIT 1;`
+- Handle requires a Data Model to be created (forseti create model)
+- Query the handle using `SELECT handle FROM forseti_security.model order by created_at_datetime desc LIMIT 1;`
 
 ```bash
+# create source.env file.  Replace the variable values with the correct values.  You'll likely need to change `CLOUDSQL_PASSWORD`, `PROJECT_ID` (`FORSETI_SERVER_VM_CHANNEL` and `FORSETI_DATA_MODEL_HANDLE` are required ONLY for IAM Explain functionality).
 cat > source.env << EOF
-export CLOUDSQL_HOSTNAME="[IP HERE]"
-export CLOUDSQL_USERNAME="[YOUR_CLOUDSQL_USER_HERE]"
-export CLOUDSQL_PASSWORD="[YOUR_CLOUDSQL_PASSWORD_HERE]"
+export API_HOST="0.0.0.0"
+export API_PORT="8080"
+export CLOUDSQL_HOSTNAME="127.0.0.1"
+export CLOUDSQL_USERNAME="root"
+export CLOUDSQL_PASSWORD=""
 export CLOUDSQL_SCHEMA="forseti_security"
-export FORSETI_SERVER_VM_CHANNEL="[VM_IP]:[GRPC_PORT]"
-export FORSETI_DATA_MODEL_HANDLE="[DATA_MODEL_HANDLE_HASH:21254f1de747879237a95cb552e80844]"
-EOF
-
-# create auth config.json for authenticating
-# refer to instructions on setting up project authentication
-cat > server/config.json << EOF
-{
-  "host": "0.0.0.0",
-  "port": 8080,
-  "bodyLimit": "100kb",
-  "corsHeaders": ["Link"],
-
-  "oauth2ClientId": "[SERVICE_ACCOUNT_NAME]@apps.googleusercontent.com",
-  "oauth2ClientSecret": "[CLIENT_SECRET]",
-  "oauth2Callback": "http://localhost:8080/auth/google/callback",
-}
+export FORSETI_SERVER_VM_CHANNEL="0.0.0.0:50051"
+export FORSETI_DATA_MODEL_HANDLE="21254f1de747879237a95cb552e80844"
```

Comment:
same here, is there a better placeholder to put here instead of putting a model id?